src/Microsoft.AspNetCore.Rewrite/Rule.cs

```diff
@@ -3,10 +3,16 @@
 
 namespace Microsoft.AspNetCore.Rewrite
 {
-    // make this public and doc comements
-    // caller must set the context.Results field appropriately in rule.
+    /// <summary>
+    /// Represents an abstract rule.
+    /// </summary>
     public abstract class Rule
     {
+        /// <summary>
+        /// Applies the rule.
+        /// Note that it is the responsibility of ApplyRule to set the context.Result appropriately.
```

Comment:
Suggested change of wording: "Implementations of ApplyRule should set a value for RewriteContext.Result."